components/movementsensor/movementsensor.go

```diff
@@ -130,32 +130,32 @@ func Readings(ctx context.Context, g MovementSensor) (map[string]interface{}, er
 	readings := map[string]interface{}{}
 
 	pos, altitide, err := g.Position(ctx)
-	if err != nil {
+	if err != nil && err.Error() != ErrMethodUnimplementedPosition.Error() {
```

Comment:
nit, i think it's more idiomatic to do this

```suggestion
	if err != nil && errors.Is(err, ErrMethodUnimplementedPosition) {
```